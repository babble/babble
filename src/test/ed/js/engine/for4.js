
print( "before" );
a = null;
for ( var i in a ){
    print( i );
}
print( "after" );


print( "before" );
a = { z : 1 };
for ( var i in a ){
    print( i );
}
print( "after" );
