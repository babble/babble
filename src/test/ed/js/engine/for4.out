before
after
before
z
after
